server/src/test/java/org/apache/iotdb/db/mpp/execution/operator/AggregationOperatorTest.java

```diff
@@ -108,8 +109,18 @@ public void testAggregateIntermediateResult1() throws IllegalPathException {
     AggregationOperator aggregationOperator =
         initAggregationOperator(aggregationTypes, null, inputLocations);
     int count = 0;
-    while (aggregationOperator.hasNext()) {
+    while (true) {
+      ListenableFuture<?> blocked = aggregationOperator.isBlocked();
+      if (!blocked.isDone()) {
+        continue;
+      }
```

Comment:
```suggestion
      blocked.get();
```
hang up the thread instead of making it busy running.

Reply:
fixed